Documentation/configuration/metrics.rst

```diff
@@ -42,6 +42,12 @@ Datapath
 --------
 
 * ``datapath_errors_total``: Total number of errors occurred in datapath management, labeled by area, name and address family.
+* ``datapath_conntrack_gc_expired_total``: Number of deleted entries in the conntrack table by the garbage collector process
+* ``datapath_conntrack_gc_runs_total``: Number of times that the conntract garbage collector process was run
+* ``datapath_conntrack_gc_incomplete_runs_total``: Number of times that the conntrack garbage collector failed
+* ``datapath_conntrack_gc_key_fallbacks``: Number of times that the Key fallback was invalid.
```

Comment:
Is this the number of times we had 1 bad key and used the fallback? or when that fallback was also bad?
If it's the second one, does that trigger a restart of the GC? Would it be better to record conntrack_gc_restart_total and then us a label for the reason? This would only be useful if we have other reasons to restart, or if we wanted to establish a convention around metrics like this for other GC/cleanup things.

Reply:
This is the number of successful key fallbacks. Any successful or unsuccessful GC run may have 0..n key fallbacks. The interrupted counter is for when we restart